### ISSUE TITLE
Gyro Notch Filtering

### ISSUE THREAD

**Issue by superjax** [rosflight/firmware#296]
Another feature from the FPV community that could be useful - notch filtering gyro measurements. (another suggestion from @korken89)

I just went perusing betaflight (I should probably do this more often), and I noticed that they are basically using biquad filters on all their axes.  They have some other smarts in there, the coefficients on the biquad filters appear to be dynamic - perhaps some adaptive notch adjustment based on the FFT of the gyro data?

@drinkdhmo pointed out to me the other day that at 8kHz, we should totally be able to observe a lot of the noise in the motors, this would be a neat way to do it.

**Comment by korken89** [comment]
Indeed! I can do a small write up on this as well when I make the one for ESC telemetry :)
